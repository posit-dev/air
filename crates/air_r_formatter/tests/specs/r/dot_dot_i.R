..1

..10

function(...) {
  ..1 + ..2
}
