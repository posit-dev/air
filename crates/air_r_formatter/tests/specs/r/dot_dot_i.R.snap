---
source: crates/air_formatter_test/src/snapshot_builder.rs
info: r/dot_dot_i.R
---
# Input

```R
..1

..10

function(...) {
  ..1 + ..2
}

```


=============================

# Outputs

## Output 1

-----
Indent style: Space
Indent width: 2
Line ending: LF
Line width: 80
Magic line break: Respect
-----

```R
..1

..10

function(...) {
  ..1 + ..2
}
```
